doc/source/operations.rst

```diff
@@ -192,6 +192,39 @@ to
 
    heat_api: "{{ suse_osh_registry_location }}/openstackhelm/heat:stein"
 
+Updating OpenStack Service Configuration
+----------------------------------------
+
+Certain use cases may require the addition or modification of OpenStack service
+configuration parameters. To update the configuration for a particular service,
+parameters can be added or modified in the 'conf' section of that service's chart.
+For example, to change the logging level of the Keystone service to 'debug', locate
+the 'conf' section of the Keystone chart located at 
+socok8s/site/soc/software/charts/osh/openstack-keystone/keystone.yaml and add the
+following lines, beginning with the 'logging' key:
+
+.. code-block:: yaml
+
+   conf:
+     logging:
+       logger_root:
+         level: DEBUG
+       logger_keystone:
+         level: DEBUG
+
+.. note::
+
+   Please consult the documentation for the appropriate OpenStack service for
```

Comment:
could we add an example what to consult, in this case, the openstack-helm/keystone/values.yaml.

Reply:
maybe add a link to https://docs.openstack.org/rocky/configuration/index.html